jumpcloud-ADMU/Powershell/Form.ps1

```diff
@@ -558,6 +693,11 @@ $tbJumpCloudUserName.add_TextChanged( {
             $img_localaccount_valid.Source = DecodeBase64Image -ImageBase64 $ActiveBase64
             $img_localaccount_valid.ToolTip = $null
         }
+        if($tbJumpCloudUserName.Text -eq $hostname){
+            Write-ToLog "JumpCloud Username can not be the same as the hostname"
+            $script:bMigrateProfile.IsEnabled = $false
```

Comment:
![Screenshot 2024-07-12 at 2 49 53 PM](https://github.com/user-attachments/assets/a7533bdc-f5e1-409c-a3ca-d89758b1e15d)
Thanks for validating this in form + start migration :)